nav2_bt_navigator/README.md

```diff
@@ -24,11 +24,13 @@ A Behavior Tree consists of control flow nodes, such as fallback, sequence, para
 
 ## Navigation Behavior Trees
 
-The BT Navigator package has two sample XML-based descriptions of BTs.  These trees are [navigate_w_replanning.xml](behavior_trees/navigate_w_replanning.xml) and [navigate_w_replanning_and_recovery.xml](behavior_trees/navigate_w_replanning_and_recovery.xml).  The user may use any of these sample trees or develop a more complex tree which could better suit the user's needs.
+The BT Navigator package has three sample XML-based descriptions of BTs.  
+These trees are [navigate_w_replanning_time.xml](behavior_trees/navigate_w_replanning_time.xml), [navigate_w_replanning_distance.xml](behavior_trees/navigate_w_replanning_distance.xml) and [navigate_w_replanning_and_recovery.xml](behavior_trees/navigate_w_replanning_and_recovery.xml).  
+The user may use any of these sample trees or develop a more complex tree which could better suit the user's needs.
 
-### Navigate with Replanning
+### Navigate with Replanning (time-based)
```

Comment:
May want to add an analog section for the distance based so that its clear why its differentiated..